Directory.Build.props

```diff
@@ -16,4 +16,8 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="$(MicrosoftNetCompilersToolsetPackageVersion)" PrivateAssets="All" />
```

Comment:
How long is this needed for? Add a TODO for removing it.

Reply:
Probably until the next VS release.

Reply:
Ok. Add a todo and :shipit:

This is semi blocking me (and anyone else using VS) at the moment.